gtk/src/toga_gtk/app.py

```diff
@@ -34,14 +34,6 @@ def create(self):
         icon_impl = toga_App.app.icon._impl
         self.native.set_icon(icon_impl.native_72.get_pixbuf())
 
-    def set_app(self, app):
-        super().set_app(app)
-
-        # The GTK docs list set_wmclass() as deprecated (and "pointless")
-        # but it's the only way I've found that actually sets the
-        # Application name to something other than '__main__.py'.
-        self.native.set_wmclass(app.interface.name, app.interface.name)
-
```

Comment:
This keeps getting reported as a bug, even though it's clearly labeled a warning; since it's functionally broken on Ubuntu 22.04 (although it still works on Fedora 36) anyway, it seems better to remove the deprecated call and find a different approach.